roles/keystone-setup/tasks/main.yml

```diff
@@ -34,6 +34,7 @@
   keystone_user: tenant={{ item }}
                  tenant_description="{{ item }} tenant"
                  token={{ admin_token }}
+                 endpoint="{{ endpoints.keystone.url.port.admin }}/{{ endpoints.keystone.version }}/" 
```

Comment:
I'm a little confused. You changed the defaults in the `keystone_user` module (and a few other modules), and we changed the ports the service listens on, but now we need to provide an argument to this module? Did you change the default to something we aren't listening on? Previously the default was the port we were listening on.

Also, previously, the default was to connect to localhost (http) rather than through haproxy (https). The change here to pass along an `endpoint` argument changes this to connect through haproxy (https). Is this an intended change of behavior?